Source/Core/DolphinWX/Debugger/MemoryCheckDlg.h

```diff
@@ -5,6 +5,8 @@
 #pragma once
 
 #include <wx/dialog.h>
+#include <wx/radiobut.h>
```

Comment:
wxRadioButton and wxStaticText can be forward declared (just like wxCheckBox is).